armi/tests/tutorials/c5g7-blueprints.yaml

```diff
@@ -1,56 +1,167 @@
+        # U235 density is based on Macros of U02 for fission XS in group 3. Group 3
+        # was chosen because it is below the U-238 fast fission cutoff, but also the
+        # U-235 fission cross section is not high enough the self shielding will play
+        # a large roll. This assuming U235 is the only fissioning isotope. Assuming FC
+        # is ~90% enriched (90.9% by atom but wanted only 1 sig fig).
+        # Had very similar macro absorption XS as guide tube, so using same
+        # Al and water with just a little U.
+        input format: number densities
+        H: 6.70e-2
+        O: 3.35E-2
+        B: 2.87E-5
+        U235: 6e-8
+        U238: 6e-9
 # end-custom-isotopics
 blocks:
     uo2: &block_uo2
         grid name: UO2 grid
         fuel:
             shape: Circle
             material: UO2
+            isotopics: UO2
             Tinput: 25.0
             Thot: 25.0
-            od:  1.08
+            od: .8190
+            latticeIDs: [U]
+        void 1: &fuel_gap_1
+            shape: Circle
+            material: Void
+            Tinput: 25.0
+            Thot: 25.0
+            id: fuel.od
+            od: zirconium clad.id
+            latticeIDs: [U]
+        zirconium clad: &clad_Zr
+            shape: Circle
+            material: Custom
+            isotopics: Zr clad
+            Tinput: 25.0
+            Thot: 25.0
+            id: .8360
+            od: .9500
+            latticeIDs: [U]
+        void 2: &fuel_gap_2
+            shape: Circle
+            material: Void
+            Tinput: 25.0
+            Thot: 25.0
+            id: zirconium clad.od
+            od: aluminum clad.id
+            latticeIDs: [U]
+        aluminum clad: &clad_Al
+            # NEA/NSC/DOC(96)2 Documents:
+            # "This clad is used to simulate hot conditions at room temperature
+            # (decrease the moderation ratio)"
+            shape: Circle
+            material: Custom
+            isotopics: Al clad
+            Tinput: 25.0
+            Thot: 25.0
+            id: .9700
+            od: 1.0800
             latticeIDs: [U]
         moderator: &moderator
             shape: DerivedShape
             material: SaturatedWater
+            isotopics: moderator
             Tinput: 450.0
             Thot: 450.0
+        inner moderator GT: &guide_tube_moderator
+            shape: Circle
+            material: SaturatedWater
+            isotopics: moderator
+            Tinput: 25.0
+            Thot: 25.0
+            od: guide tube.id
+            latticeIDs: [GT]
         guide tube: &guide_tube
-            shape: Square
+            shape: Circle
             material: Custom
-            isotopics: GuideTube
+            isotopics: Al clad
             Tinput: 25.0
             Thot: 25.0
-            widthOuter: 1.26
+            id: .6800
+            od: 1.0800
             latticeIDs: [GT]
+        FC GT: &fission_chamber_guide_tube
```

Comment:
I recommend spelling out all guide tubes and fission chambers to be consistent.

Reply:
this is a fags problem. I didn't want the the fission chamber tube to have the fission chamber flag or the guide tube flag.